test/e2e/flowaggregator_test.go

```diff
@@ -1261,7 +1275,10 @@ func deployDenyNetworkPolicies(t *testing.T, data *TestData, pod1, pod2 string)
 		t.Errorf("Error when creating Network Policy: %v", err)
 	}
 	// Wait for NetworkPolicy to be realized.
-	if err := WaitNetworkPolicyRealize(2, data); err != nil {
+	if err := WaitNetworkPolicyRealize(openflow.IngressRuleTable, antreaIngressTableInitFlowCount+1, data); err != nil {
```

Comment:
Thanks Wenying for fixing this. 
Here we are creating k8s NP deny rules, are we supposed to check by the `ingressTableInitFlowCount`? seems to be a typo? same for below

Reply:
Thanks for poining it out, I would change it.